templates/base.html

```diff
@@ -30,6 +30,24 @@
   </head>
 
   <body>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light container">
```

Comment:
We don't want it to display on larger screens, so I think we could add the class `d-md-none` so hide it once it is no longer a small screen.

Reply:
ok a will add to the header